test.js

```diff
@@ -1,5 +1,12 @@
 var t = "hello";
 
-if (t === "hello') {
+var selec = '[data-dikkemik;
+
+
+if (t == "hello') {
```

Comment:
__[jshint]__
*  Missing semicolon.

*  Expected an identifier and instead saw ')'.

*  Expected an assignment or function call and instead saw an expression.